scripts/oneoff/swap-supplier-duns.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+"""
+This script attempts to swap the DUNS numbers for two suppliers. This involves temporarily setting one of the suppliers
+to a fake DUNS number to satisfy the database's uniqueness constraint.
+
+Usage: swap-supplier-duns.py <supplier_1> <supplier_2> <duns_1> <duns_2> [options]
+
+    <supplier_1>                                          Supplier ID #1
+    <supplier_2>                                          Supplier ID #2
+    <duns_1>                                              DUNS number #1
+    <duns_2>                                              DUNS number #2
+
+Options:
+    --stage=<stage>                                       Stage to target
+    --updated-by=<updated-by>                             Updater email address for audit trail
+    --dummy-duns=<dummy-duns>                             Dummy DUNS number
```

Comment:
You could specify the default dummy DUNS number here, which might make it easier to find

```suggestion
    --dummy-duns=<dummy-duns>                             Dummy DUNS number [default: 0000000002]
```